enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -32,7 +38,7 @@ class Meta:
     )
     active = models.BooleanField(default=False)
     group_uuid = models.UUIDField(null=True, blank=True, db_index=True)
```

Comment:
I believe `null` should be set to `False` because for now we will always receive an `enterprise_customer_uuid` and we will treating that as `group_id`